op-service/client/ethclient.go

```diff
@@ -16,3 +21,50 @@ func DialEthClientWithTimeout(ctx context.Context, url string, timeout time.Dura
 
 	return ethclient.DialContext(ctxt, url)
 }
+
+const BatcherFallbackThreshold int64 = 10
+const ProposerFallbackThreshold int64 = 3
+const TxmgrFallbackThreshold int64 = 3
```

Comment:
Op-batcher and op-proposer do not have metrics like `op_node_default_rpc_client_responses_total` for reference, but when I tested locally, I found that 10 sometimes cannot trigger the fallback well, especially Txmgr, because it periodically submits transactions, and the request frequency is very low. Do you have any better suggested values?